quadtree/src/lib.rs

```diff
@@ -214,9 +214,95 @@ impl fmt::Display for NodeId {
     }
 }
 
+pub enum SpatialNeighbor {
```

Comment:
nit: how about renaming this `enum` to `Direction` or `SpatialNeighborKind`?

Reply:
Done, I renamed it to `Direction`.